lib/Ed25519VerificationKey2020.js

```diff
@@ -31,10 +39,24 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
     super(options);
     this.type = SUITE_ID;
     this.publicKeyMultibase = options.publicKeyMultibase;
+    this.privateKeyMultibase = options.privateKeyMultibase;
+
     if(!this.publicKeyMultibase) {
       throw new TypeError('The "publicKeyMultibase" property is required.');
     }
-    this.privateKeyMultibase = options.privateKeyMultibase;
+
+    if(!this.publicKeyMultibase || !_isValidKeyHeader(
+      this.publicKeyMultibase, MULTICODEC_ED25519_PUB_HEADER)) {
+      throw new Error('`publicKeyMultibase` has invalid byte header: ' +
+      `"${this.publicKeyMultibase}".`);
+    }
+
+    if(this.privateKeyMultibase && !_isValidKeyHeader(
+      this.privateKeyMultibase, MULTICODEC_ED25519_PRIV_HEADER)) {
+      throw new Error('`privateKeyMultibase` has invalid byte header: ' +
+       `"${this.privateKeyMultibase}".`);
```

Comment:
Stylistically, the param should be dropped to the next line (here and above). It may also happen to all fit on a single line when doing so as an added benefit.

Reply:
Fixed in 2db6cc7ae7988dde2660d438b538777442a1c67a. Dropping it to the next line doesn't allow everything to fit on one line (would've done it otherwise). The error now takes 3 lines... don't agree that it's any more or less readable than the two line version.